crates/forge/tests/it/zk/factory_deps.rs

```diff
@@ -44,9 +45,9 @@ contract ZkLargeFactoryDependenciesScript is Script {
 
     let node = ZkSyncNode::start();
 
-    // foundry default gas-limit is not enough to pay for factory deps in our current
-    // default environment
-    let gas_limit = u32::MAX >> 1;
+    // foundry default gas-limit is not enough to pay for factory deps
+    // with era_test_node's environment
```

Comment:
```suggestion
    // with Anvil-zksync's environment
```
For communication purposes to be in sync with https://github.com/matter-labs/era-test-node